src/VisualStudio/Core/Def/Implementation/AbstractOleCommandTarget.Execute.cs

```diff
@@ -882,6 +886,21 @@ private void ExecuteFindReferences(ITextBuffer subjectBuffer, IContentType conte
                 lastHandler: executeNextCommandTarget);
         }
 
+        protected virtual void SyncClassView(SnapshotPoint caretPosition)
+        {
+        }
+
+        void ExecuteSyncClassView(ITextBuffer subjectBuffer, IContentType contentType, Action executeNextCommandTarget)
```

Comment:
Because that involved a bunch of extra code for something that I can't see us wanting in a non-VS host.